holidayextras/holidayextras.github.com
# Added syntax highlighting to code blocks

#### What does this PR do?
Adds syntax highlighting to code blocks within our posts.
#### What unit or integration tests does this PR have?
None
#### What selenium tests does this PR have?
None
#### How should a developer review this?
Check that the changes i've made make sense, i'm not 100% sure on making a new html file to include a script tag. What do others think?
#### How should this be manually tested?
Run this branch locally and visit a post like harleys express tutorial, you should see all the code snippets have been highlighted.
#### Any background context you want to provide?
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### What gif best describes how you feel about this work?
![](http://media.giphy.com/media/3ONE2VRuBE7Ze/giphy-tumblr.gif)
---
#### Quality checklist (for PR author to complete BEFORE code review):

#### IP or Developer Review:

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
